jest.config.ts

```diff
@@ -6,6 +6,16 @@ const createJestConfig = nextJest({ dir: "./" });
 const jestConfig: Config = {
   preset: "ts-jest",
   testEnvironment: "jsdom",
+  // uncomment later after adding tests 
```

Comment:
let's mimic the current coverage that the repo has. That way nothing that pushes the coverage below it can be committed.

Reply:
Cool. Understood